raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -275,8 +291,8 @@ def clear_if_finalized(iteration, channelidentifiers_to_channels):
 def next_channel_from_routes(
         available_routes: List['RouteState'],
         channelidentifiers_to_channels: Dict,
-        transfer_amount: int,
-        lock_timeout: int,
+        transfer_amount: typing.PaymentAmount,
+        lock_timeout: typing.BlockTimeout,
 ) -> NettingChannelState:
```

Comment:
Please fix the type here, it's `typing.Optional[NettingChannelState]`

Reply:
yeah already done locally. I am running mypy :)